layerstack/layer.py

```diff
@@ -416,17 +417,25 @@ class Layer(object):
         Directory from which to load the layer
     """
 
-    def __init__(self, layer_dir):
+    def __init__(self, layer_dir, model=None):
```

Comment:
Is it straightforward to have the model get passed in from the command line? (When you are running a single layer?) It looks like we probably aren't doing that?